Makefile

```diff
@@ -132,3 +132,7 @@ bundle: kustomize
 .PHONY: bundle-build
 bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+
+# TODO Replace with controller_gen
+manifests: $(KUSTOMIZE)
+	$(KUSTOMIZE) build config/crd > config/crd/bases/konk.infoblox.com_konks.yaml
```

Comment:
Does this replace the kustomize template with the kustomize rendered yaml?

Reply:
the configuration is in config/crd/kustomization.yaml. config/crd/bases/*.yaml are created from that config

Reply:
Maybe this should write to `helm-charts/konk-operator/crds/`

Reply:
hmm it deleted the file now... wut